DigitalExcellence/dex-frontend
# There is no silent-refresh.html, redirect to auth-callback instead

## Description

Current we are trying to silent refresh using the silent-refresh.html, this does not exist though. The silent refresh can make a call to auth-callback, which does exist.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Open Digital Excellence and log-in
2. Wait for your token to expire, watch the tab and check if it calls for a refresh token

## Link to issue

Closes: #253
